idact/detail/add_cluster_app/main.py

```diff
@@ -29,11 +29,21 @@
               type=int,
               help="The ssh port. Default: 22")
 @click.option('--auth',
-              default='PUBLIC_KEY',
+              default='GENERATE_NEW_KEY',
               type=str,
               help="Authentication method. "
-              "Avilable values: PUBLIC_KEY, ASK_EVERYTIME. "
-              "Default: PUBLIC_KEY")
+              "Available values: GENERATE_KEY, PRIVATE_KEY, ASK_EVERYTIME. "
+              "Default: GENERATE_KEY")
+@click.option('--key',
+              default="~/.ssh",
+              type=str,
+              help="Path to the previously generated private key "
+                   "Default location: ~/.ssh")
+@click.option('--key_type',
```

Comment:
For now, when only RSA is supported, this option makes no sense. I know that that is for future improvement, but it wouldn't be hard to provide user with this option when other key types would be supported. Or do you think that it should stay anyway?